.github/workflows/jni_artifacts.yml

```diff
@@ -21,12 +21,14 @@ jobs:
 
     strategy:
       matrix:
-        os: [windows-latest, macos-latest]
+        os: [windows-latest, macos-latest, ubuntu-latest]
```

Comment:
This could create hard dependencies on GLIBC_2.38?

Reply:
We use the rust toolchain, which I believe has the glibc set to 2.17+ at the moment.